orchestra/utils/task_lifecycle.py

```diff
@@ -589,6 +589,7 @@ def tasks_assigned_to_worker(worker):
                 'detail': task_assignment.task.project.short_description,
                 'priority': task_assignment.task.project.priority,
                 'state': state,
+                'start_datetime': task_assignment.task.project.start_datetime,
```

Comment:
This is the project's `start_datetime` but the column is called `Task Created`. `task` and `task.assignemnt` also have a `start_datetime` for what it's worth, and I suspect an expert's mental model is around when they joined the project (the task assignment `start_datetime`) or the overall task's start rather than whenever the project started.

Reply:
whatever you decide, name the property appropriately: `project_start_datetime`/`task_start_datetime`/`assignment_start_datetime`

Reply:
Done! Great point- I picked `assignment_start_datetime`